rewrite-xml/src/main/java/org/openrewrite/xml/AutoFormat.java

```diff
@@ -15,41 +15,36 @@
  */
 package org.openrewrite.xml;
 
-import org.openrewrite.ExecutionContext;
-import org.openrewrite.Tree;
 import org.openrewrite.xml.search.FindIndentXml;
 import org.openrewrite.xml.tree.Xml;
 
-import java.util.List;
 import java.util.stream.Collectors;
 
 import static java.util.Arrays.stream;
 import static java.util.stream.IntStream.range;
 
-public class AutoFormat extends XmlProcessor<ExecutionContext> {
+public class AutoFormat<P> extends XmlProcessor<P> {
```

Comment:
For another PR perhaps, but I think `AutoFormat` should be promoted to a `Recipe`?